src/sentry/api/endpoints/project_details.py

```diff
@@ -509,11 +516,24 @@ def get(self, request: Request, project) -> Response:
         if "hasAlertIntegration" in expand:
             data["hasAlertIntegrationInstalled"] = has_alert_integration(project)
 
-        if features.has(
-            "organizations:dynamic-sampling-basic", project.organization, actor=request.user
-        ):
-            data.pop("dynamicSampling", None)
+        ds_feature_multiplexer = DynamicSamplingFeatureMultiplexer(project, request)
 
+        # Dynamic Sampling Logic
+        if ds_feature_multiplexer.is_on_dynamic_sampling():
```

Comment:
if you apply my other suggestion you can use as property:
```suggestion
        if ds_feature_multiplexer.is_on_dynamic_sampling:
```